protocol/deputy-pause-module.md

```diff
@@ -0,0 +1,108 @@
+could fill a whole role of toilet paper and are not just limited to the issues noted above.
+
+## Proposed Solution
+
+We propose eliminating the pre-signed pause system entirely by introducing a `DeputyPauseModule` to
+be installed into the primary Foundation Safe. The `DeputyPauseModule` would allow a deputy account
+to act as the Foundation Safe to trigger the Superchain-wide pause functionality. The deputy
+account would not have any other capabilities.
+
+The `DeputyPauseModule` solves a litany of problems. Because the deputy account could be made to be
+an externally owned account, we gain the benefit of the pre-signed pause (immediate pause
+execution) without all of the many downsides of a pre-signed transaction. We could also remove the
+Foundation Operations Safe entirely, generally simplifying our multisig setup.
+
+### EOA vs Smart Contract
+
+We propose using an Externally Owned Account (EOA) instead of a smart contract as the deputy here.
+Using an EOA is simpler and easier to reason about. The private key for this EOA can be stored
+securely and made accessible to a limited set of security personel.
+
+### Single Account vs Mapping
+
+We propose having the `DeputyPauseModule` use a single account instead of a mapping of accounts
+that are able to act as the deputy. A single account is easier to keep track of and having multiple
+accounts does not decrease the risk involved with this module, it simply spreads it across more
+private keys. Having multiple keys be able to act as the deputy here might have some slighty
+benefits but this begins to scope creep beyond the original intention of replacing the pre-signed
+pause functionality.
+
+### Exact Proposed Modifications
+
+We currently have two Safe accounts for the Foundation:
+
+1. `0x847B5c174615B1B7fDF770882256e2D3E95b9D92` - known as the "Foundation Upgrade Safe" was
+  deployed on 2024-02-08 and is used as one owner on the 2/2 System Owner Safe that can upgrade
+  OP Mainnet (alongside the Security Council).
+2. `0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A` - known as the "Foundation Operations Safe" was
+  deployed on 2021-01-16, is used for pre-signed pauses, and is one signer on the 2/2 System Owner
+  Safe that can upgrade Base.
+
+We propose the following actions:
+
+1. `DeputyPauseModule` will be installed into `0x9BA6`.
+1. `0x847B` will be replaced with `0x9BA6` in the OP Mainnet 2/2.
+1. `0x847B` will be deprecated generally.
+
+Please note that step (2) requires action by the Security Council and governance communication. If
+going through the whole process of replacing one of the 2/2 signers is too much of a pain, we can
+also consider the following alternative:
+
+1. `DeputyPauseModule` will be installed into `0x9BA6`.
+1. `DeputyPauseModule` will be installed into `0x847B`.
+1. Security Council sets `0x847B` as the Deputy Guardian.
+1. Base replaces `0x9BA6` with `0x847B` in its 2/2.
+1. `0x9BA6` is deprecated generally.
```

Comment:
The top flow feels cleaner to me. The big risk with either of these flows is changing one owner on the 2/2, as we can brick it if we transfer to the wrong owner. The options there are either:
1. A single `swapOwner` call. Risk is messing up the `swapOwner` call and bricking the safe
2. Add `0x9BA6` as an owner so it's a 2/3, ensure we can execute a tx, then remove the original `0x847B`. Downside is 3 transactions we need to execute, and there is a period where Foundation can unilaterally upgrade

Reply:
Tagged in Ben. Personally I feel like changing the Deputy Guardian is much safer than touching the 2/2 but open to be challenged on that.

Reply:
In either scenario (step 2 of the top flow, or step 4 of the bottom alternate flow) we do have to change owners on a 2/2 and will have the same decision between `swapOwner` and a three step process where the Foundation temporarily has unilateral control. The difference is just which 2/2 it is

Reply:
Yeah I guess you're correct on that point!